### DISCUSSION TITLE
Option::<Option<T>>::Some(None) How do I use this ? It is not clear from the documentation.

### DISCUSSION THREAD

**Discussion by Abhicodes-crypto** [diesel-rs/diesel#3324] [Q&A] [answered]
For setting a value of a field to None. They have said to use ```Option::<Option<T>>::Some(None) ``` in https://docs.diesel.rs/2.0.x/diesel/prelude/derive.AsChangeset.html But What do I name the field type then. `Option::Option<String>> ` gives error. If I name it as `Option<String> `. Then type mismatch . PLease help me.

**Comment by Mingun** [comment]
`Option<Option<T>>`

**Comment by Abhicodes-crypto** [comment, marked as answer]
In the schema it should be `Nullable<Type>`. If it is not you will get the above issue. And the type `Option<Option<T>>`. Should be used for AsChangeSet derivable.
